### DISCUSSION TITLE
关于断网续传需要验证问题

### DISCUSSION THREAD

**Discussion by coldpub** [spencerwooo/onedrive-vercel-index#1013] [Q&A]
想问一下，当下载端网速不好导致下载几个G大小的文件时，例如用idm下载到后面链接不上了 再链接提示需要输入登陆信息才能继续，这个时候输入网址绑定的onedrive账号不行，请问一下如何解决突然断网后的续传！

**Comment by spencerwooo** [comment]
这个是 OneDrive 提供的直链 API 由于时间太长导致过期失效了的问题，应该无法从本项目这边进行解决。

**Comment by coldpub** [comment]
我用idm暂时解决了这个问题（重新获取新的下载链接 在idm里面替换原来的下载链接，可以续传
Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
